webpack/scenes/ContentViews/Details/Versions/ContentViewVersionEnvironments.js

```diff
@@ -18,7 +18,7 @@ const ContentViewVersionEnvironments = ({ environments }) => {
           <Label color="purple" href={`/lifecycle_environments/${env.id}`}>{`${env.name}`}</Label>
         </FlexItem>
         <FlexItem>
-          <InactiveText text={` ${env.publish_date} ago`} /><br />
```

Comment:
This was adding some space b/w the environment labels. Should we add some spacing here?

Reply:
I'll review it's usage and make sure of this.

Reply:
<img width="325" alt="Screen Shot 2021-11-12 at 1 43 00 PM" src="https://user-images.githubusercontent.com/38083295/141532100-8c3aaa75-ad73-43b2-9825-377a0ec346b9.png">

Spacing added